### DISCUSSION TITLE
Max lenght of character in field

### DISCUSSION THREAD

**Discussion by willy89pl** [Security-Onion-Solutions/securityonion#13789] [Unsupported Versions]
Hi all,

I have a problem with character length limit.

My SO receives syslog (with JSON inside), processed by ingest pipeline. It works and is fine.

But every now and then a single line in incoming logs is very long. >65536 characters (2^16, this could be a good clue) and I noticed that it does not fit in the message field (it is truncated, cuted).

I did some research, tried set "message" as "wildcard" type, set "character length limit" to 100000 characters, but it does not work.

Has anyone had a similar problem, maybe solved it?

**Comment by reyesj2** [comment]
Is that intended as a single continuous message? Is it possible to break the syslog data up into chunks? I imagine having a message field that long is going to limit how you can make use of it. If possible I would look at setting up an ingest pipeline to break the message into key:values that make sense for you. https://docs.securityonion.net/en/2.4/elasticsearch.html#parsing
